frontend/src/components/Board/PostDetail/PostHtmlView.module.css

```
@@ -0,0 +1,19 @@
+.content {
+  width: 100%;
+  max-width: var(--width-content);
+  margin-bottom: 40px;
+  color: var(--color-text-dark);
+  line-height: 1.8;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)